src/racer/scopes.rs

```diff
@@ -228,6 +228,7 @@ pub fn get_start_of_search_expr(src: &str, point: usize) -> usize {
             (b'.', State::MustEndsWithDot(_)) =>  State::None,
             (b':', State::MustEndsWithDot(index)) =>  State::StartsWithCol(index),
             (b':', State::StartsWithCol(_)) =>  State::None,
+            (b'?', State::StartsWithDot) => State::None,
```

Comment:
As written, this will only work for method or field accesses on a `Try` operator. Should `?` be in [`util::is_search_expr_char`](https://github.com/phildawes/racer/blob/master/src/racer/util.rs#L14) instead?